packages/slice-machine/tests/src/modules/screenshots/utils.ts

```diff
@@ -28,7 +27,7 @@ export const getSelectedSliceDummyData = () => {
     extension: "js",
     model: dummyModel,
     screenshots: {},
-    mock: mocks as unknown as SharedSliceContent[],
+    mock: mocks as unknown as ComponentMocks,
```

Comment:
this is where property based testing would come in handy :)

Reply:
but actually we could runtime validate the mocks here to type them properly and at least you're sure that's the right format for your tests maybe.
But yeah I guess your suggestion makes sense. 
That's something to consider for the future